stable/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -85,9 +85,9 @@ spec:
         - name: csi-resizer
           image: "{{ .Values.image.csiResizer.repository }}:{{ .Values.image.csiResizer.tag }}"
           args:
-            - "-csi-address=$(ADDRESS)"
-            - "-v=5"
-            - "-leader-election"
+            - -csi-address=$(ADDRESS)
```

Comment:
doesn't it need to be double-dash ?

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade

Reply:
I will ignore the inconsistency of the also in upstream existing arguments 🤣 ... for now 😉